mautic/mautic
# Add 191 length validation on lookup type contact field

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no




#### Description:
Every contact lookup type field is VARCHAR(191). Contact creation and imports were crashing when trying to set longer values.
This PR fixes it.



#### Steps to test this PR:
1. Create lookup type field
2. Create a new contact with a too long text field (>191 chars) to the field
3. See that an error is displayed
4. Import fail also with long text

Credits @Damzoneuh




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10225"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
